### PR TITLE
Update all dependencies and flatten deps tree.

### PR DESCRIPTION
**Pull request by XhmikosR**
569 packages -> 320 packages

### CODE REVIEW

**Comment by XhmikosR** [comment]
@bryanbraun this will pass after #112 is merged and this is rebased.

**Comment by XhmikosR** [comment]
@bryanbraun just don't leave this hanging for a long time because deps will have changed by the time you merge it

**Comment by XhmikosR** [comment]
@bryanbraun friendly ping :) I can't keep this up to date indefinitely

**Comment by bryanbraun** [comment]
@XhmikosR, ahh, sorry for the delay.

These PRs are very helpful! I was waiting for a good time when I could knock all of these out. I should be able to dedicate some time tonight. 👍

**Comment by XhmikosR** [comment]
NP, I just don't want it to get outdated :)

BTW only the replacement of phantomjs is left, but I'll let you handle that later. It does still work, but it's deprecated.

File: package-lock.json [hunk 1]
```diff
@@ -692,45 +394,41 @@
...
-      "dev": true,
-      "optional": true
+      "version": "2.20.0",
+      "resolved": "https://registry.npmjs.org/commander/-/commander-2.20.0.tgz",
+      "integrity": "sha512-7j2y+40w61zy6YC2iRNpUe/NwhNyoXrYpHMrSunaMG64nRnaf96zO/KMQR4OyN/UnE5KLyEBnKHd4aG3rskjpQ==",
+      "dev": true
     },
     "component-bind": {
       "version": "1.0.0",
-      "resolved": "http://artifactory.gapinc.dev/artifactory/api/npm/npm-repos/component-bind/-/component-bind-1.0.0.tgz",
```
Comment (XhmikosR) [inline package-lock.json]: I'm not sure how you ever got these...

I suggest that you clean install the latest Node.js LTS, and do `npm cache clean -fg`. Pruning local modules wouldn't hurt.

**Comment by XhmikosR** [comment]
I'd also suggest that you enable [automated security fixes](https://help.github.com/en/articles/configuring-automated-security-fixes) in your repos.

**Comment by bryanbraun** [comment]
Thanks! Will do.

**Pull request merged**